avahi-sharp/Makefile.am

```diff
@@ -73,10 +73,10 @@ monodoc_DATA = avahi-sharp-docs.zip avahi-sharp-docs.tree avahi-sharp-docs.sourc
 endif
 
 install-data-hook: $(ASSEMBLY)
-	$(AM_V_GEN)MONO_SHARED_DIR=. $(GACUTIL) /i $(ASSEMBLY) /package avahi-sharp /gacdir $(libdir) /root $(DESTDIR)$(libdir)
+	$(AM_V_GEN)MONO_SHARED_DIR=. $(GACUTIL) /i $(ASSEMBLY) /package avahi-sharp /gacdir $(prefix)/lib /root $(DESTDIR)$(prefix)/lib
```

Comment:
Is this path the same on Debian/Ubuntu and Fedora/RHEL families? Wouldn't configure time flag to override this path make more sense?

Reply:
I would use variable defined at the top, ``sharplib=$(prefix)/lib, then use it multiple times. If anyone needs to override it, it would make it easier.

Reply:
Verified both Fedora and Debian use the same path, it should work for both. I think it is reasonable to expect it would work for most other distributions as well.

Reply:
How did you check that? The Debian avahi package does not build the mono bindings?